fixtures/pages-functions-with-routes-app/tests/index.test.ts

```diff
@@ -42,14 +42,18 @@ describe("Pages Functions with custom _routes.json", () => {
 		});
 	});
 
-	it.concurrent("should render static pages", async () => {
-		await readyPromise;
-		const response = await fetch(`http://${ip}:${port}/undefined-route`);
-		const text = await response.text();
-		expect(text).toContain(
-			"Bienvenue sur notre projet &#10024; pages-functions-with-routes-app!"
-		);
-	});
+	it.concurrent(
+		"should render static pages",
+		async () => {
+			await readyPromise;
```

Comment:
Could this file use the `runWranglerPagesDev` helper as well?

Reply:
Yes it could but I got the tests to pass and had fatigue on updating them any further... 😓 
I think we could probably pull that function definition out to a shared place but wasn't keen to get into that here.